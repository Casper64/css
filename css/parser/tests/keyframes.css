
@keyframes {}

@keyframes test }

.test {
    color: red;
}

@keyframes next {
    from
    to {}
    not_from {}
    10%
    20% {}
    60% {}
    .test {
        
    }
}

.color {
    background-color: red;
}

.color2 {
    background-color: red;
}