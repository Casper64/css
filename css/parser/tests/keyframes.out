css/parser/tests/keyframes.css:2:12: error: expecting the name of the keyframes rule
    1 | 
    2 | @keyframes {}
      |            ^
    3 | 
    4 | @keyframes test }
css/parser/tests/keyframes.css:4:17: error: expecting "{"
    2 | @keyframes {}
    3 | 
    4 | @keyframes test }
      |                 ^
    5 | 
    6 | .test {
css/parser/tests/keyframes.css:12:5: error: expecting "{"
   10 | @keyframes next {
   11 |     from
   12 |     to {}
      |     ~~
   13 |     not_from {}
   14 |     10%
css/parser/tests/keyframes.css:13:5: error: expecting a percentage or "from" or "to"
   11 |     from
   12 |     to {}
   13 |     not_from {}
      |     ~~~~~~~~
   14 |     10%
   15 |     20% {}
css/parser/tests/keyframes.css:15:5: error: expecting "{"
   13 |     not_from {}
   14 |     10%
   15 |     20% {}
      |     ~~
   16 |     60% {}
   17 |     .test {
css/parser/tests/keyframes.css:17:5: error: expecting a keyframes rule
   15 |     20% {}
   16 |     60% {}
   17 |     .test {
      |     ^
   18 |         
   19 |     }
css/parser/tests/keyframes.css:20:1: error: expecting a css selector
   18 |         
   19 |     }
   20 | }
      | ^
   21 | 
   22 | .color {